Examples/iOSExplorer/iOSExplorer/ODXItemCollectionViewController.m

```diff
@@ -87,7 +87,9 @@ - (void)signInAction
         if (!error){
             self.client = client;
             [self loadChildren];
-            self.navigationItem.rightBarButtonItem = self.actions;
+            dispatch_async(dispatch_get_main_queue(), ^(){
```

Comment:
Yes, none of the callbacks happen on the main thread and you can only update UI from the main thread.